aiidalab_widgets_base/viewers.py

```diff
@@ -348,8 +348,20 @@ def _observe_cell(self, _=None):
                 spglib_structure, symprec=1e-5, angle_tolerance=1.0
             )
 
+            periodicity_map = {
```

Comment:
Nice! :+1:

Reply:
@danielhollas  Thank you!,  I will include the  test